api_call_options.go

```diff
@@ -37,6 +37,9 @@ type GetOptions struct {
 	// their `Href` field.
 	Excludes []string `url:"exclude,omitempty,comma"`
 
+	// Arbitrary params for API URL, used for arbitrary filters
+	ArbitraryOpts map[string]string `url:"-"`
```

Comment:
I don't love this name, but I can't think of anything markedly better. `QueryParams` (Parameters?) perhaps.

In any case, let's up the description line to start with the name of the field. Go linters tend to want the documentation to start with the exact name of the field.

Reply:
Will do.